src/update.fs

```diff
@@ -0,0 +1,186 @@
+
+module UpdateCmd
```

Comment:
I think what I meant was "please take the time to record what this stuff is actually doing" :). I know the original cmd files weren't documented (and of course they should have been) - but now you're going through all this with such a fine toothcomb, it's a great opportunity to add a few basic comments recording what you've learned.